docs/conf/portland/2020/sponsors/online-prospectus.rst

```diff
@@ -0,0 +1,171 @@
+We've expanded the offerings and have introduced a lot of new things,
+but please let us know if there are other points of interaction with our community that would be valuable for you.
+
+Concept
+=======
+
+{% include "conf/sponsorship/concept.rst" %}
+
+Demographics
+============
+
+{% include "conf/sponsorship/demographics.rst" %}
+
+Why Sponsor
+===========
+
+{% include "conf/sponsorship/why.rst" %}
+
+Sponsorship Packages
+====================
+
+The following options are suggested sponsorship levels. We are happy to discuss adjustments and custom packages.
+
+.. warning:: This is a draft prospectus and is subject to change.
+
+Second Draft
+------------
+
+The **Second Draft** package is great for companies looking to hire or promote a product.
+
+- **Increased** Three (3) tickets_
+- **Increased** Two (2) featured job postings on our Job Board, also promoted in our newsletter ({{ newsletter_subs }} subscribers)
+- **Increased** A short description and logo (200 words) of your company on the conference website
+- A table at our **virtual job fair**
+- Name included in welcome announcement in email newsletters and social media
+
+The **Second Draft** package costs **{{sponsorship.second_draft.price}}**.
+
+Publisher
+---------
+
+The **Publisher** package is great for sending a team and getting to know the community.
+
+- **Increased** Seven (7) tickets_
+- **Increased** Four (4) featured job postings on our Job Board, also promoted in our newsletter ({{ newsletter_subs }} subscribers)
+- **Increased** A short description and logo (200 words) of your company on the conference website
+- A table at our **virtual job fair**
+- Name included in welcome announcement in email newsletters and social media
+- **New** A small logo on all Write the Docs website pages for 3 months
+
+The **Publisher** package costs **{{sponsorship.publisher.price}}**.
+
+Patron
+------
+
+**Limit 3**
+
+The **Patron** package highlights your company as a force in the industry and community:
+
+- **Increased** Thirteen (13) tickets_
+- **Increased** Seven (7) featured job postings on our Job Board, also promoted in our newsletter ({{ newsletter_subs }} subscribers)
+- **Increased** A medium description and logo (500 words) of your company on the conference website
+- A **virtual sponsorship booth**
+- A featured table at our **virtual job fair**
+- Small logo included in intermission slides and on talk videos
+- Name included in welcome announcement in email newsletters and social media
+- **New** 5 minute sponsored lightning talk on main stage of the conference
+- **New** One newsletter sponsorship (logo & 300 words) to our newsletter ({{ newsletter_subs }}+ subscribers)
+- **New** A logo on all Write the Docs website pages until the end of 2020. (30,000 pageviews/mo)
+- **New** A :doc:`small ad </sponsorship/website>` on the Write the Docs website (240x180px, 180 characters, 10,000 pageviews/mo) for 3 months
```

Comment:
Maybe say "Write the Docs home page" instead of "Write the Docs website"? I assume that's what it means, as the pageviews are quite different.

Reply:
This would be on any of our non-conference website pages. I think more clarity is probably needed on the number being the amount you get, not the total that we receive.

Reply:
So do we want to say

> A small ad to be displayed on a single page of the WTD website, of your choice? (subject to availability)?

Reply:
No, it's across all pages in the sidebar, like the current sponsor promo.

Reply:
:+1: right